sandy/sections/mf8.py

```diff
@@ -544,8 +546,8 @@ def _write_rdd(sec):
     65146100.0 44179.7000          0          0          6          01803 8457    2
     163850.000 403.088000 1554660.00 895.993000 0.00000000 0.000000001803 8457    3
     4.00000000 1.00000000          0          0         12          21803 8457    4
-    2.00000000 0.00000000 1233300.00 800.000000 3.000000-6 1.000000-61803 8457    5
-    1.00000000 0.00000000 2058980.00 330.000000 9.999970-1 1.000000-61803 8457    6
+    20.0000000 0.00000000 1233300.00 800.000000 3.000000-6 1.000000-61803 8457    5
```

Comment:
the first number should be `2.00000000`, not `20.0000000`

Reply:
More or less... do you mean you prefer to change the rtype format here instead of in to_endf6 method? In the second example of this method there are two rtyps that should be 4.0 and 6.0 instead of 40.0 and 60.0. It was 2.0 in my previous change because I kept the "digit.digits" format ... As we have discussed, this is then corrected in the to_endf6 method which returns the value in the "digit.digits" format. If you prefer, I can correct the value of rtyp here instead of in the to_endf6 method. It depends if the _write_rdd(sec) method must have the same syntax of the section mf=8 mt=457 of the endf6 file or it must keep the information written as in sec

Reply:
yes, it must be changed in  `_write_rdd`. There is no reason for it to have a different syntax in `_write_rdd` and `to_endf6`.
What is important is consistency! if you change it in `_write_rdd` it should be fixed also for `to_endf6` because in the latter you call [write_mf8](https://github.com/EnricaBelfiore/sandy/blob/BR/sandy/decay.py#L772 ).

Also there is a critical problem if the implemented tests do not fail when they arewrong, right @AitorBengoechea ?!
Tests **MUST NOT** be changed to pass if they are not correct!